svg.panzoom.js

```diff
@@ -43,20 +41,20 @@ SVG.extend(SVG.Doc, SVG.Nested, {
     var pinchZoomStop = function(ev) {
       ev.preventDefault()
       zoomInProgress = false
-      
+
       SVG.off(document,'touchmove', pinchZoom)
       SVG.off(document,'touchend', pinchZoomStop)
       this.on('touchstart', pinchZoomStart)
-      
+
       if(ev.touches.length > 0) panStart.call(this, ev)
     }
 
     var pinchZoom = function(ev) {
       ev.preventDefault()
       var currentTouches = normalizeEvent(ev)
 
-      var lastDelta = Math.sqrt((lastTouches[0].clientX - lastTouches[1].clientX)**2 + (lastTouches[0].clientY - lastTouches[1].clientY)**2)
-      var currentDelta = Math.sqrt((currentTouches[0].clientX - currentTouches[1].clientX)**2 + (currentTouches[0].clientY - currentTouches[1].clientY)**2)
+      var lastDelta = Math.sqrt( Math.pow(lastTouches[0].clientX - lastTouches[1].clientX, 2) + Math.pow(lastTouches[0].clientY - lastTouches[1].clientY, 2) )
```

Comment:
I'm getting **Uncaught TypeError: Cannot read property 'clientX' of undefined** here on Android 4.1, sometimes. I wonder if it has to do with the option object in `SVG.on` becoming a capture boolean when options are not supported and then we're are missing some event bubbling.

But I have been debugging this a lot now and it's not 100% consistent with that. If it the case then it should also happen in modern browsers when the capture flag is set to `true`, but it does not.